clients/src/main/resources/common/message/RequestHeader.json

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 
 {
+  "apiKey": -1,
```

Comment:
Why was this needed?

Reply:
It shouldn't be needed-- I'll remove it.

In the long term, we'll want to generate other things besides ApiMessage from JSON files (enums, common structures, etc.) but we can hold off on that for now.